rlclientlib/schema/v2/Metadata.fbs

```diff
@@ -1,6 +1,6 @@
 namespace reinforcement_learning.messages.flatbuff.v2;
 
-enum PayloadType : ubyte { CB, CCB, Slates, Outcome }
+enum PayloadType : ubyte { CB, CA, CCB, Slates, Outcome }
```

Comment:
In practice we haven't released anything yet, so it does not really matter, but we probably should append new enum valuees to the end in that kind of cases.

Reply:
Appended to end